charts/operator-wandb/values.yaml

```diff
@@ -74,6 +74,17 @@ global:
     parameters: {}
     caCert: ""
 
+  kafka:
+    # The following values are anchored here, and referenced by alias later for the kafka dependency chart.
+    # List of username for kafka, first being the admin user.  Must have the same number of elements as the passwords value
+    users: &kafkaUsers
```

Comment:
Can we make this kafkaUser instead of kafkaUsers?
Reason being that this is going to be used by frfu, and there we'll need a variable with a single string that contains the user name.
Here we'll need:
```
users: &kafkaUser
passwords: &kafkaPassword "wandb"
```
Below we'll need:
```
      users:
            - *kafkaUser
      passwords: *kafkaPassword
```

Reply:
Is this necessary or can frfu be aware that it's getting a list and a comma delimited string?  This is also at the global layer so we can always add a pared down "individual" user/password value to the app specific blocks.